notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "        polynomial),  \"spline1\"  (linear  spline),  and \"spline3\" (cubic\n",
+    "        spline).  The functions may be abbreviated.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: order\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "**order**\n",
+    "        The order of the polynomials or the number of spline pieces."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: low_reject, high_reject\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Rejection limits below  and  above  the  fit  in  units  of  the\n",
+    "        residual  sigma.   Unequal  limits  are  used to reject spectral\n",
+    "        lines on one side of the continuum during continuum fitting.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: niterate\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Number of rejection iterations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: grow\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "When a pixel is rejected, pixels within  this  distance  of  the\n",
+    "        rejected pixel are also rejected.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Fitting: markrej\n",
+    "\n",
+    "*Description*:\n",
+    "\n",
+    "Mark  rejected  points?   If  there  are many rejected points it\n",
+    "        might be desired to not mark rejected points."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Arithmetic-related operations "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section deals with arithmetic between multiple spectra.\n",
+    "\n",
+    "1. In the notebook, this is trivial, as `Spectrum1D` objects support arithmetic.  E.g., subtracting a smoothed version of a spectrum from the spectrum and showing it can be done as:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.load_data(specviz.current_spectrum - box_smooth(specviz.current_spectrum, 10))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "2. In the UI, this is potentially much more complicated, as a plugin to support arithmetic would be necessary. Such a plugin is possible, but not necessarily desirable, as part experience suggests the feature requests for this quickly spiral, suggesting simply saying \"use the notebook\" is a better answer.\n",
+    "\n",
+    "    Note, though: two piece of arithmetic are critical: a way to subtract a fitted continuum and a way to divide by it.  These could be implemented as part of the *fitting* plugin, however, to avoid having to write a separate arithmetic plugin for this purpose."
+   ]
+  },
```

Comment:
If continuum subtraction and division are crucial, why not bite the bullet and create the plugin?  Alternatively there could be separate plugins for these two functions.

Reply:
My argument here is based on the Qt specviz, where once an arithmetic plugin was created, it had endless feature creep and became less useful because it was everything for everyone (which meant sort of nothing for anyone since it wasn't clear how/when to use it).  So my point is to emphasize a *different* workflow of: 
1. fit a continuum
2. subtract *or* divide by that continuum

but *not* a general add/subtract any spectrum from any other, since that's the sort of operation I think we want to push people toward the notebook for.

Now that you say it, though, that does *not* mean we can't have two separate plugins.  So I'll update the notebook to indicate that this *could* be done but isn't part of the initial spec?

Reply:
Well, glue already supports arithmetic amongst glue `Data` objects -- it may actually be much more straightforward to say "we support what glue supports right now" and just leverage that.